Eastern-Research-Group/mywaterway
# Feature/update monitoring stations labels

## Related Issues:
* None. @cschwinderg, I still don't have access to EPA's Jira, and won't until my new PIV card is ready to be picked up (hopefully in a few weeks)

## Main Changes:
NOTE: all changes below are on the Community Overview and/or the Community Monitoring tabs:
* Changed "Sample Locations" checkbox label to "Monitoring Stations" (this is the checkbox above the "Monitoring & Sensors" tab
* Changed "Daily Stream Flow Conditions" checkbox label to "Daily Water Conditions"
* Changed "Monitoring Stations" checkbox label to "Sample Locations"
* Changed "Sample Type" label to "Monitoring Type" (this is shown in the accordion/listview in the "Monitoring & Sensors" tab of the Community Overview page)
* Changed all variables backing those labels to their new respective values

## Steps To Test:
1. Navigate to http://localhost:3000/community/020700100102/overview
2. View new labels and ensure nothing broke due to the renamings
